### PR TITLE
Fix bugs, update PPP

### PR DESCRIPTION
**Pull request by Chen-Kuei**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -575,7 +575,33 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**Extensions**
+
+* 1a. The given index is invalid.
+    * 1a1. EmployeeManager informs user that the given index is invalid.
+    * 1a2. User enters new index.
+    * Steps 1a1-1a2 are repeated until the index entered is valid.
+    * Use case resumes from step 2.
+* 1b. EmployeeManager detects invalid information.
+    * 1b1. EmployeeManager informs user that some information is invalid.
+    * 1b2. EmployeeManager requests user to enter details of the employee.
```
Comment (Goh-Li-Ting) [inline docs/DeveloperGuide.md:596]: I think we discussed to remove this for the `add` use case so we should also remove for the `edit` use case

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -575,7 +575,33 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    * 1b3. User enters the requested details.
+    * Steps 1b1 - 1b3 are repeated until the data entered are correct.
```
Comment (Goh-Li-Ting) [inline docs/DeveloperGuide.md:598]: Similarly, we discussed changing "are correct" to "is correct" for the `add` use case. Perhaps to make it better, we can write "Steps 1b1 - 1b2 are repeated until the information entered is valid"?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -575,7 +575,33 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+**Use case: UC4 - Edit an employee**
+
+**MSS**
+
+1. User requests to edit an employee with index and new details.
+2. EmployeeManager edits the employee.
+3. EmployeeManager shows an updated list of employees.
+4. Use case ends.
+
+**Extensions**
```
Comment (Goh-Li-Ting) [inline docs/DeveloperGuide.md:587]: An extension for the scenario where _no details inputted_ seems to be missing

File: docs/images/EditCommandSequenceDiagram.png
Comment (Goh-Li-Ting) [inline docs/images/EditCommandSequenceDiagram.png:1]: Perhaps we should remove the parsing section of the sequence diagram and just show from `execute`? Similar to the sequence diagrams used for `increment` and `clear` as the parsing section was already explained in the logic component

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

**Review by aexolate — APPROVED** [review]
LGTM

**Pull request merged**